docs/getting-started/version-12.md

```diff
@@ -0,0 +1,70 @@
+A wind of change with Highcharts version 12
+
+Highcharts v12 has introduced new Webpack-based UMD packages to make Highcharts modules easier and faster.
+Thanks to this change everyone can soon also use the full potential of ESM packages.
+The new UMD is with some setups compatible to the previous UMD, while it also brings some changes for other setups.
+This document lays out the changes of Highcharts v12 packages.
+
+
+
+1. No module factory
+
+The new UMD does not come with a factory for Highcharts modules anymore.
+Instead of calling each factory with the Highcharts namespace, all modules will recognize automatically the shared namespace.
+This means less code to load Highcharts modules.
+
+Before v12:
+```js
+import Highcharts from 'highcharts';
+import HighchartsExport from 'highcharts/modules/export';
+HighchartsExport(Highcharts);
+```
+
+With v12:
+```js
+import Highcharts from 'highcharts';
+import 'highcharts/modules/export';
+```
+
+Support both variants:
+```js
+import Highcharts from 'highcharts';
+import HighchartsExport from 'highcharts/modules/export';
+typeof HighchartsExport === 'function' && HighchartsExport(Highcharts);
+```
+
+
+
+2. Asynchronous / Lazy loading requires ESM
+
+Because of the nature of UMD, async support for Highcharts modules was limited to certain setups only.
+The new UMD is now consistently expecting synchronous loading in all setups.
+For an asynchronous approach, one needs to use the ESM variants, which can be found in the `highcharts/es-modules/masters/` path.
+
+* HTML Example:
+  ```js
+  <script type="module">
+  import Highcharts from 'https://code.highcharts.com/highcharts/es-modules/masters/highcharts.js';
+  import 'https://code.highcharts.com/highcharts/es-modules/masters/highcharts-more.js';
+  import 'https://code.highcharts.com/highcharts/es-modules/masters/modules/offline-exporting.js';
```

Comment:
Won't loading offline-exporting without exporting loaded create any issues? If no, that would nice improvement worth extra info in the docs. If yes, exporting should be loaded before.